scripts/canflash/cantransfer.sh

```diff
@@ -0,0 +1,39 @@
+#Check if all arguments have been passed
+if [ "$#" -lt 3 ]; then
+    echo "ERROR enter all arguments: sh cantransfer.sh 'bin_files' 'pi_username' 'IP_ADDRESS' 'destination_path'"
+    exit 1
+fi
+
+BIN_FILE=$1
+PI_USERNAME=$2
+IP_ADDRESS=$3
+
+# If destination path is set as an argument; it will be set as the destination path
+# Otherwise it is set as the PI's home directory
+# If the destination path is a file, the path will be the PI's home directory
+if [ -d "3SM4" ]; then
+  DESTINATION_PATH=$4
+elif [ -f "3SM4" ]; then
+  DESTINATION_PATH="/home/$PI_USERNAME"
+else
+  DESTINATION_PATH="/home/$PI_USERNAME"
+fi
```

Comment:
No need for `home/username` as the script can only transfer files within this directory. There will be a permission error if you try to transfer a file to a directory that isn't under `home/username`. For `scp`, it is assumed the destination directory is relative to `/home/username`.

i.e. `sh cantransfer.sh test.txt username IP new-directory` will transfer the file to `home/username/new-directory/test.txt`